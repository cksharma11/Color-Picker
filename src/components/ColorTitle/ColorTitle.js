import React from "react";

const ColorTitle = ({ colorTitle }) => {
  return <div className="div-color-title">{colorTitle}</div>;
};

export default ColorTitle;
