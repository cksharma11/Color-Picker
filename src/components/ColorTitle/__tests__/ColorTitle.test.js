import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ColorTitle from "../ColorTitle";

Enzyme.configure({ adapter: new Adapter() });

describe("ColorTitle", () => {
  it("should render properly", () => {
    const wrapper = shallow(<ColorTitle colorTitle="red" />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
