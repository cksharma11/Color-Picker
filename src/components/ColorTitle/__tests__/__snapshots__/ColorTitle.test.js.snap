// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorTitle should render properly 1`] = `
"<div className=\\"div-color-title\\">
  red
</div>"
`;
